lib/config/generator.js

```diff
@@ -0,0 +1,210 @@
+var path = require('path');
+var Vow = require('vow');
+var vowNode = require('vow-node');
+var Table = require('cli-table');
+var prompt = require('prompt');
+var fs = require('fs');
+
+var Checker = require('../checker');
+var utils = require('../utils');
+
+prompt.message = '';
+prompt.delimiter = '';
+prompt.start();
+
+/**
+ * Script that walks the user through the autoconfig flow
+ * @type {String[]}
+ */
+var prompts = [
+    {
+        name: 'Please choose a preset number:'.green,
+        require: true,
+        pattern: /[0-9]*/
```

Comment:
I think we should allow custom presets somehow. It will be useful feature if we allow to reconfigure already existed `.jscsrc` according to new code if rules were not specified in `.jscsrc` but were in preset. Something like auto-nullifier?